src/guide/advanced/isi.md

```diff
@@ -44,11 +44,12 @@ part of the original `TRUSTED_PEERS` array to the network.
 
 <!-- Check: a reference about future releases or work in progress -->
 
-Registering an account is different. Iroha can be compiled in two modes:
-_public_ and _private_. If it's compiled with _private_ permissions, to
-register an account, you need an account. This is the default. If you want
-your users to be able to register without access to a pre-existing account,
-you need to compile Iroha in the _public_ mode.
+Registering an account is different. Iroha can be compiled in two
+[modes](../configure/modes.md): _public_ and _private_. If it's compiled
+with _private_ permissions, to register an account, you need an account.
+This is the default. If you want your users to be able to register without
+access to a pre-existing account, you need to compile Iroha in the _public_
+mode.
```

Comment:
No longer valid. 

```suggestion
Depending on how you decide to set up your `genesis.json` the process for registering an account might go one of two ways. But first, a little about permissions. Iroha by default allows all instructions to go through, until a permission validator has been registered which would restrict it. Some permission validators that we thought would be useful for `private` and some for `public` blockchain use cases can be added to your genesis block, however in that case, the registration process for an account is different. 

In a _public_ blockchain anyone should be able to register an account[^1]. So in theory all that you need is a suitable client, a way to generate a private key of a suitable type (ED25519) and that's it. 

In a _private_ blockchain, however, the situation is a little different. Each private blockchain has their own unique process for allowing `Register<Account>` instructions. The provided in-house permission validators for the private use-case only need for the new account to be registered by an existing account, and any account can register any other account. However, by definition, a private blockchain can have any process for setting up an account: it could be that the registering instruction has to be submitted by a specific account, or by a smartcontract that asks for other details. It could be that in a private blockchain registering new accounts is only possible on specific dates, or limited by a non-mintable (finite) token. And even public blockchains can have private domains, where weird rules apply. We cannot cover all the possible processes, so instead in this section we focus on the default meant to represent a _typical_ private blockchain.  

[^1]: In fact, once we have finished with our key-centric address scheme for accounts, you don't register an account as much as claim it. 
```

Reply:
I've reviewed this and split it between ISI page and the page on private/public modes